kotest-framework/kotest-framework-multiplatform-reporter/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+   id("kotest-multiplatform-library-conventions")
+   id("kotest-android-native-conventions")
+   id("kotest-watchos-device-conventions")
+   id("kotest-native-conventions")
+   id("kotest-js-conventions")
+   id("kotest-publishing-conventions")
+   //TODO, once AGP is working: id("com.android.library")
```

Comment:
if we don't get this to work, then we need an ugly check in the JVM target to see whether we're running on Android or JVM (for returning the correct target name). Android native is not affected by this.